src/components/AddInput/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from "react";
+import { inline } from "./style.css";
+
+interface IAddInput {
```

Comment:
FYI It depends on the project, however generally, Typescript naming convention doesn't use the `I` prefix
For react props you can use something like `ComponentNameProps` instead